sound/soc/sof/sof-priv.h

```diff
@@ -98,6 +98,12 @@ enum sof_debugfs_access_type {
 	SOF_DEBUGFS_ACCESS_D0_ONLY,
 };
 
+struct sof_compr_stream {
+	uint32_t copied_total;
+	uint32_t sampling_rate;
+	size_t posn_offset;
```

Comment:
what is the definition for posn_offset? confused with the first path removing the use of posn_offset.

Reply:
posn_offset represents the index inside the 'stream_box' mailbox where the information `sof_ipc_stream_posn` is held (e.g dai_posn, host_posn, etc).

I have previously removed this line:

```
      tstamp->byte_offset = sdev->stream_box.offset + ipc_params_reply.posn_offset;
```

because byte_offset is totally different thing from posn_offset. byte_offset is the offset in the ring buffer to DSP holding actual audio data, while posn_offset is an index in a shared memory area holding audio information.

Reply:
actually posn_offset is not even used in this patch series. will be added later. So to remove confusion, I think it is best to have it removed from here and add it when it is used with proper explanation.

Reply:
@plbossart I removed `posn_offset` from this series. as it is not used. Will add it with the next series.